ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -259,7 +259,8 @@ public void init() {
         "/metacard/:id",
         (req, res) -> {
           String id = req.params(":id");
-          return util.metacardToJson(id);
+          String storeId = Optional.ofNullable(req.queryParams("storeId")).orElse(null);
```

Comment:
Why are you using an Optional? It looks like you are saying "If this value is null, then return null".

Reply:
We used this for patch so I assumed it was a safeguard against getting an NPE if that param didn't exist, but I'm not super familiar with how optional params work.

Reply:
I don't think the use Optional is doing anything for you.

Reply:
The `Optional.ofNullable(req.queryParams("storeId"))` returns a populated Optional if `req.queryParams("storeId")` is non-null. It returns an Optional.empty() otherwise. The `orElse` extracts the value in the Optional, or returns null if the Optional is empty. You can replace the entire thing with `req.queryParams("storeId")`